core/scheduler/scheduler.go

```diff
@@ -446,7 +444,7 @@ func resolveActiveValidators(ctx context.Context, eth2Cl eth2Provider,
 	var resp []validator
 	for index, val := range vals {
 		if !val.Status.IsActive() {
-			log.Debug(ctx, "skipping inactive validator", z.U64("index", uint64(index)))
+			log.Info(ctx, "skipping inactive validator", z.U64("index", uint64(index)))
```

Comment:
This will spam each time it fetches validators

Reply:
only if validator is inactive. i think i should remove this